ORStools/gui/ORStoolsDialog.py

```diff
@@ -281,7 +281,10 @@ def run_gui_control(self):
             )
             return
 
-        clnt = client.Client(provider)
+        filename = os.path.basename(__file__).split(".")[0]
+        agent = f"QGis_{filename}"
+
+        clnt = client.Client(provider, agent)
```

Comment:
This is the only place where we're issuing a GUI-Request, correct?
If so, I'd hardcode the agent and not mess with filename-stuff…

Reply:
Done